.github/workflows/enzyme-ci.yml

```diff
@@ -55,3 +52,36 @@ jobs:
       run: cd enzyme/build && make check-activityanalysis -j`nproc`
     - name: make check-enzyme
       run: cd enzyme/build && make check-enzyme -j`nproc`
+  
+  build-macos:
+    name: Enzyme CI LLVM ${{ matrix.llvm }} ${{ matrix.build }} macOS
+    runs-on: macos-latest
+    
+    strategy:
+      fail-fast: false
+      matrix:
+        llvm: ["8", "9", "11", "12", "13"] #, "14"]
+        build: ["Release", "Debug"] # "RelWithDebInfo"
```

Comment:
Same fetch depth and llvm comment?

Reply:
Brew does not offer llvm 7 and 10 for some reason.